proxyd/backend.go

```diff
@@ -340,91 +328,19 @@ func (b *Backend) Forward(ctx context.Context, reqs []*RPCReq, isBatch bool) ([]
 		return res, err
 	}
 
-	b.setOffline()
 	return nil, wrapErr(lastError, "permanent error forwarding request")
 }
 
 func (b *Backend) ProxyWS(clientConn *websocket.Conn, methodWhitelist *StringSet) (*WSProxier, error) {
-	if !b.Online() {
-		return nil, ErrBackendOffline
-	}
-	if b.IsWSSaturated() {
```

Comment:
I'm fine removing this for now, but I do see a world where we may need the ability to limit concurrent WS connections. MEV bots will make thousands of connections to a single upstream to try and spam requests.

Reply:
I'm sure we'll have to do some refactoring in WS code when we re-enable it. At that time, we can move to an simple atomic integer or the new sliding window impl to track the current number of conns.